nano/lib/numbers.cpp

```diff
@@ -61,6 +62,16 @@ std::string nano::public_key::to_account () const
 	return result;
 }
 
+nano::public_key::public_key () :
+	uint256_union{ 0 }
+{
+}
+
+const nano::public_key & nano::public_key::null () const
```

Comment:
The function that creates a null public key is not used anywhere other than the equality function.
Is that expected?

Reply:
Yes, the PR provides the support, we will change occurrences (to nullptr accounts) as we see fit in other PRs.